### PR TITLE
updated kd app image versions and added a new cr yaml

### PR DESCRIPTION
**Pull request by ginochen**
updated bluedata images:
- kd-notebook:1.14
- kd-api-serving:1.4 (for deployment) 
- kd-mlflow:1.2

added a cr for mlflow

### CODE REVIEW

**Comment by kmathur2** [comment]
Please resolve merge conflicts

**Review by kmathur2 — APPROVED** [review]

File: deploy/example_clusters/cr-cluster-mlflow.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: "kubedirector.hpe.com/v1beta1"
+kind: "KubeDirectorCluster"
+metadata:
+  name: "mlflow-instance"
+  namespace: "aiml"
```
Comment (kmathur2) [inline deploy/example_clusters/cr-cluster-mlflow.yaml:6]: namespace should not be aiml

File: deploy/example_clusters/cr-cluster-mlflow.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    secrets:
+      - mlflow-sc
+  roles:
+    -
+      id: "controller"
+      members: 1
+      resources:
+        requests:
+          cpu: "1"
+          memory: "1Gi"
```
Comment (kmathur2) [inline deploy/example_clusters/cr-cluster-mlflow.yaml:22]: is 1 Gi enough?

**Comment by ginochen** [comment]
@Kartik ok added to 4Gi

**Pull request merged**